Plugins/Gradle/src/main/kotlin/com/shopify/testify/internal/GradleProjectExtensions.kt

```diff
@@ -54,5 +54,10 @@ val Project.inferredAndroidTestInstallTask: String?
 
 val Project.inferredDefaultTestVariantId: String
     get() {
-        return this.android.testVariants.minBy { it.testedVariant.flavorName }?.applicationId ?: ""
+        val testVariant = this.android.testVariants.minByOrNull { it.testedVariant.flavorName }
+        return try {
+            testVariant?.applicationId
+        } catch (e: Throwable) {
+            "${testVariant?.testedVariant?.applicationId}.test"
```

Comment:
DM'd about this not working - but just reading over https://issuetracker.google.com/issues/194911958#comment10 it would appear it's resolved in upcoming 7.0.1. Regarding the regressions within the 7.1.0 alphas, other issues have raised the issue as well.

I'm in favour of falling back to an empty string and requiring setting it explicitly until it's fixed upstream. Getting a new beta cut will unblock people wishing to use Gradle 7 / AGP 7.0.